actions/global_actions.jsx

```diff
@@ -107,6 +107,14 @@ export function emitChannelClickEvent(channel) {
     }
 }
 
+export function updateLastViewedChannel(channelId) {
+    return () => dispatch({
+        type: ActionTypes.UPDATE_CHANNEL_LAST_VIEWED_AT,
+        channel_id: channelId,
+        last_viewed_at: Date.now(),
+    });
```

Comment:
The dispatch and extra function here are unnecessary. Since this doesn't need `getState`, dispatch multiple actions, or do anything asynchronous, it can just returns a plain object redux action.
```
return {
    type: ActionTypes.UPDATE_CHANNEL_LAST_VIEWED_AT,
    channel_id: channelId,
    last_viewed_at: Date.now(),
};
```